__tests__/login/oidc/OidcLoginHandler.spec.ts

```diff
@@ -89,4 +93,30 @@ describe("OidcLoginHandler", () => {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     await expect(handler.canHandle({} as any)).resolves.toBe(false);
   });
+
+  it("should clear the local storage when logging in", async () => {
+    const nonEmptyStorage = mockStorageUtility({
+      someUser: { someKey: "someValue" },
+    });
+    nonEmptyStorage.setForUser(
+      "someUser",
+      { someKey: "someValue" },
+      { secure: true }
+    );
+    const handler = getInitialisedHandler({
+      storageUtility: nonEmptyStorage,
+    });
+    handler.handle({
+      sessionId: "someUser",
+      oidcIssuer: new URL("https://arbitrary.url"),
+      redirectUrl: new URL("https://app.com/redirect"),
+      clientId: "coolApp",
+    });
+    expect(nonEmptyStorage.getForUser("someUser", "someKey", { secure: true }))
+      .toBeUndefined;
+    expect(nonEmptyStorage.getForUser("someUser", "someKey", { secure: false }))
+      .toBeUndefined;
+    // This test is only necessary until the key is stored safely
+    expect(nonEmptyStorage.get("clientKey", { secure: false })).toBeUndefined;
```

Comment:
I don't think these actually assert anything right now :P Might be worth enabling the Jest plugin in ESLint if that isn't enabled yet, as I think that will warn you of tests without any assertions.

```suggestion
    expect(nonEmptyStorage.getForUser("someUser", "someKey", { secure: true }))
      .toBeUndefined();
    expect(nonEmptyStorage.getForUser("someUser", "someKey", { secure: false }))
      .toBeUndefined();
    // This test is only necessary until the key is stored safely
    expect(nonEmptyStorage.get("clientKey", { secure: false })).toBeUndefined();
```

Reply:
Ah good catch, that was actually coming from another test I had written for logout, both fixed in b3a44615fc30e195d9423a8414a929ba7ec1adc1